### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by listen001-star**
修改了代码中文

### CODE REVIEW

File: mcr.cpp [hunk 1]
```diff
@@ -8,7 +8,7 @@ int main() {
     std::cout << "请输入第一个数字: ";
     std::cin >> num1;
 
-    std::cout << "请输入第一个数字: ";
```
Comment (kefan1242) [inline mcr.cpp:11]: little confused

**Review by kefan1242 — COMMENTED** [review]
reviewer fk

File: mcr.cpp (hunk 1 above)
Comment (lyw2) [reply mcr.cpp:11]: Incorrect serial number entered

Comment (listen001-star) [reply mcr.cpp:11]: have finished

**Comment by shaddocklove** [comment]
num3

File: mcr.cpp [hunk 2]
```diff
@@ -18,4 +18,4 @@ int main() {
     std::cout << "两个数字的和是: " << sum << std::endl;
 
     return 0;
-}
\ No newline at end of file
+}
```
Comment (lyw2) [inline mcr.cpp:21]: There may be something wrong with this scope.

File: mcr.cpp [hunk 3]
```diff
@@ -18,4 +18,4 @@ int main() {
     std::cout << "两个数字的和是: " << sum << std::endl;
```
Comment (shaddocklove) [inline mcr.cpp:18]: the direction of arrow is opposite

File: mcr.cpp [hunk 4]
```diff
@@ -18,4 +18,4 @@ int main() {
...
 
     return 0;
-}
```
Comment (pseukohhh) [inline mcr.cpp:21]: SOMETHING WRONG! The result might be wrong.

**Pull request merged**